packages/react-ui-core/src/Form/RangeSlider.js

```diff
@@ -33,14 +33,21 @@ export default class RangeSlider extends Component {
       ...props
     } = this.props
 
+    const header = props.formatHeader ?
+      props.formatHeader(this.state.value.min, this.state.value.max)
```

Comment:
it appears this component can either take a single numeric value, or this object:
```javascript
value: {
  min: 0,
  max: 10,
}
```

I'm also seeing a prop warning in the JS console:
```
warning.js:36 Warning: Failed prop type: Invalid prop `value` of type `object` supplied to `RangeSlider`, expected `number`.
    in RangeSlider
```

So, a couple things need to be done here. The `propTypes` for `value` needs to be changed to accept an object, and this code needs changed to know about the single number vs min/max `value` prop/state. Or maybe just not show the header when there is a single value? I'm not sure how much sense it makes to display a header for a single value slider.

It would also be nice to have a unit test for `formatHeader`.